ruby/person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(name, age = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] Based don the project requirement, the constructor parameters that should be optional are ` name` and `parent_permission` but you are making `age` optional. Kindly fix this.


![image](https://user-images.githubusercontent.com/84156714/231319492-b02dc67c-052c-4d7c-94b5-ffad891b5983.png)